anago

```diff
@@ -486,6 +488,15 @@ EOF
     logrun -s git commit -am \
               "Add/Update $CHANGELOG_FILE for $RELEASE_VERSION_PRIME." \
      || return 1
+    logecho -n "Move the $RELEASE_VERSION_PRIME tag: "
+    logrun -s git tag -f $RELEASE_VERSION_PRIME
```

Comment:
Doesn't this mean the tag will now point to a different commit than the hash we just baked into the build? I feel like that could lead to confusion or problems if people have come to rely on them matching (e.g. for sanity checks). It seems like the only way to preserve this consistency would be to commit relnotes before doing the build.

Reply:
Yes, I believe you are correct. :(  Unfortunately this is a chicken/egg issue where generating and publishing the release notes requires the binaries.  So we may just have to live with the release notes landing as a follow-up commit post-tag.